### PR TITLE
Ys auto versioning

### PR DESCRIPTION
**Pull request by isatrio**

### CODE REVIEW

**Comment by isatrio** [comment]
Created a bash script to release an update. 2 packages added `replace-in-file` and `standard-version`. The updater uses both to update the version of the plugin. It's using regex to replace strings in the main file plugin.

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "wp-dm-custom-embed",
-  "version": "1.2.1",
+  "version": "1.5.0-1",
```
Comment (arryanggaputra) [inline package.json:3]: so, this is autoversioning?

Comment (isatrio) [reply package.json:3]: For that not yet. I still manually set it up. It needs a base version before actually using it.

Comment (arryanggaputra) [reply package.json:3]: I think there's a command such as `npm version patch` I forget.

**Pull request merged**